### ISSUE TITLE
Grid headers are not focusable/interactable. Cannot sort column with keyboard navigation.

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-angular#2199]
### Description  
When using keyboard navigation to navigate inside the grid the user is unable to focus and interact with the headers and the icons inside those headers. This breaks the accessibility.

 * igniteui-angular version: all
 * browser: all

### Steps to reproduce  

1.  Open a grid with some sortable columns or filtering or both
2.  Press tab to navigate and space bar to interact with the header sorting or the action icons inside

### Result  
there is no visible indication that the header is focused and you cannot interact with the header or the action icons inside

### Expected result  
The user should be able to interact with the header sorting and action icons inside using keyboard navigation

### Hint
The header focus and interaction should behave the same way as the chip component inside the grid.
See the attached video.

### Attachments  
https://drive.google.com/file/d/1PyZyhbNKY9EJYOLsA_DZUx3TpF3DVOp8/view

**Comment by StefanIvanov** [comment]
@ddincheva is this issue still reproducible, have we applied any fixes to resolve it with the many improvements we made to the grid keyboard navigation?

**Comment by zdrawku** [comment]
Part of the latest alpha version - https://github.com/IgniteUI/igniteui-angular/releases/tag/9.1.0-alpha.11
